lib/rspec/expectations/syntax.rb

```diff
@@ -41,12 +41,12 @@ def enable_should(syntax_host=default_should_host)
 
         syntax_host.module_exec do
           def should(matcher=nil, message=nil, &block)
-            ::RSpec::Expectations::Syntax.warn_about_should_unless_configured(__method__)
+            ::RSpec::Expectations::Syntax.warn_about_should_unless_configured(Kernel.__method__)
```

Comment:
This should probably use `::Kernel` instead of `Kernel`.  Inside a basic object, you need to fully qualify constants:

```
irb(main):003:0> class Foo < BasicObject
irb(main):004:1> Kernel
irb(main):005:1> end
NameError: uninitialized constant Foo::Kernel
        from (irb):4:in `<class:Foo>'
        from (irb):3
        from /Users/myron/.rubies/ruby-2.1.5/bin/irb:11:in `<main>'
```

```
irb(main):012:0> class Foo < BasicObject
irb(main):013:1> ::Kernel
irb(main):014:1> end
=> Kernel
```